labelbox/schema/user.py

```diff
@@ -35,3 +38,69 @@ class User(DbObject):
     organization = Relationship.ToOne("Organization")
     created_tasks = Relationship.ToMany("Task", False, "created_tasks")
     projects = Relationship.ToMany("Project", False)
+    org_role = Relationship.ToOne("OrgRole", False)
+
+    def update_org_role(self, role: Role):
+        """ Updated the `User`s organization role. 
+
+        See client.get_roles() to get all valid roles
+        If you a user is converted from project level permissions to org level permissions and then convert back, their permissions will remain for each individual project
+
+        Args:
+            role (Role): The role that you want to set for this user.
+
+        """
+        user_id_param = "userId"
+        role_id_param = "roleId"
+        query_str = """mutation SetOrganizationRolePyApi($%s: ID!, $%s: ID!) { 
+            setOrganizationRole(data: {userId: $userId, roleId: $roleId}) { id name }}
+        """ % (user_id_param, role_id_param)
+
+        self.client.execute(query_str, {
+            user_id_param: self.uid,
+            role_id_param: role.uid
+        })
+
+    def remove_from_project(self, project: Project):
+        """ Removes a User from a project. Only used for project based users.
+        Project based user means their org role is "NONE"
+
+        Args:
+            project (Project): Project to remove user from
+
+        """
+        self.upsert_project_role(project, self.client.get_roles()['NONE'])
+
+    def upsert_project_role(self, project: Project, role: Role):
+        """ Updates or replaces a User's role in a project.
+
+        Args:
+            project (Project): The project to update the users permissions for
+            role (Role): The role to assign to this user in this project.
+        
+        """
+        org_role = self.org_role()
+        if org_role.name.upper() != 'NONE':
```

Comment:
is the value `NONE` being returned by the backend?

Reply:
Yes